### ISSUE TITLE
PDF formatting broken with pdftohtml 0.40

### ISSUE THREAD

**Issue by cawka** [kohler/hotcrp#75]
After some amount of debugging why PDF format checking giving me an error, I realized that (at least version 0.40 that I installed from Homebrew on my Mac) pdftohtml automatically adds .xml to the specified name of the xml output.

For example, after running 

    pdftohtml -xml sample.pdf sample.xml

it produced `sample.xml.xml` and not the requested `sample.xml`.  To fix this, `src/banal` script needs to be properly updated.

**Comment by kohler** [comment]
You should be using the pdftohtml that comes with poppler, as the README says.

**Comment by cawka** [comment]
Thanks!  You're right, it is my error. I saw that poppler's version is needed, but for some random reason assumed that the separately packaged version (I tried on both FreeBSD and OSX) is the poppler's one.

In any case.  After I installed the correct version of pdftohtml (for FreeBSD it would be part of `poppler-utils`, for Homebrew it would be just `poppler`), it works as expected.

Not really related to this issue, but a small thing not deserving separate issue.  `src/banal` script assumes that perl is installed into `/usr/bin`, which is not the case on FreeBSD.
